modules/howtos/pages/analytics-using-sdk.adoc

```diff
@@ -59,7 +59,8 @@ The analytics service provides an array of options to customize your query. The
 | `lcb_cmdanalytics_statement(command, statement, statement length )` | Sets the actual statement to be executed.
 | `lcb_cmdanalytics_scope_name(command, scope name, scope length)` | Associate scope name with the analytics query.
 | `lcb_cmdanalytics_named_param(command, argument name, name length, argument value, value length)` | Sets a named argument for the analytics query.
-| `lcb_cmdanalytics_positional_param(command, argument value, argument length)` | Adds a positional argument for the analytics query.
+| `lcb_cmdanalytics_positional_param(command, arguments as JSON string, argument string length)` | Adds positional arguments for the analytics query. _(From C SDK 3.2.1)_
```

Comment:
lcb_cmdanalytics_positional_param**s**

Reply:
yup, I'd managed to mis-edit `named_param` above, and must have over-tidied, thanks!
Updated now, pushed.